src/projectInitializer.ts

```diff
@@ -176,6 +176,7 @@ export class ProjectInitializer {
       boardItemList.push({
         name: board.name,
         id: board.id,
+        fqbn: board.fqbn,

```

Comment:
it's Arduino special

Reply:
I will remove the workaround and use a new class `ArduinoBoard` to make it more clean.